client/client_test.go

```diff
@@ -806,12 +805,14 @@ func runClientScan(numRows, numVersions int, b *testing.B) {
 	b.ResetTimer()
 
 	b.RunParallel(func(pb *testing.PB) {
-		keyBuf := append(make([]byte, 0, 64), []byte("key-")...)
-		endKey := []byte("key.")
+		startKeyBuf := append(make([]byte, 0, 64), []byte("key-")...)
+		endKeyBuf := append(make([]byte, 0, 64), []byte("key-")...)
+		// endKey := []byte("key.")
 		for pb.Next() {
 			// Choose a random key to start scan.
 			keyIdx := rand.Int31n(int32(numKeys - numRows))
-			startKey := proto.Key(encoding.EncodeUvarint(keyBuf[0:4], uint64(keyIdx)))
+			startKey := proto.Key(encoding.EncodeUvarint(startKeyBuf[0:4], uint64(keyIdx)))
```

Comment:
leave out the 0 if buffer slices start from the beginning: `startKeyBuf[:4]`

I think one of gofmt's modes makes this change automatically.